Mithril update 2024-08-28 (#450)
IntersectMBO/cardano-updates: blog/2024-08-28-mithril.md

---
title: Mithril Team Update
slug: 2024-08-28-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> The Mithril team has started working on decentralizing the signature orchestration of the Mithril network with a preliminary phase which moves signer registrations to the epoch settings route of the aggregator. They also worked on optimizing the memory usage of the signer when signing the Cardano transactions. Additionnally, the team has completed the implementation of a proof-of-concept for diffusing Mithril signatures with the Cardano network layer.
> 
> Finally, they added the support for retrieving database snapshots from local files in the client, and they fixed a bug on the certification of the Cardano stake distribution.

Reviewer: ```suggestion
The Mithril team started working on decentralizing the signature orchestration of the Mithril network. In this preliminary phase, they are moving signer registrations to the aggregator's epoch settings route. They also optimized memory usage for signers when signing Cardano transactions and completed a proof of concept for diffusing Mithril signatures with the Cardano network layer.

Finally, they added support for retrieving database snapshots from local files in the client and fixed a bug related to the certification of Cardano stake distribution.
```
